README.md

```diff
@@ -44,3 +44,25 @@ by communicating with the Linux kernel through `/dev/spidev*`. This means:
 
 If you modify `spi.pyx`, make sure to set the `USE_CYTHON` environment variable before building---otherwise your
 changes will not be compiled into `spi.c`.
+
+#### Running the code on Linux desktop
+
+Note that the cython code will not run on Windows.    
+
+With pip only install `pillow`, do not install `RPi.GPIO` (it is only for the Pi, on desktop Linux it will just exit with an error).
+
+Then install some dependencies:
+
+```
+sudo apt-get install python3-dev
+sudo apt-get install python3-tk
```

Comment:
This dependency has been removed now unless you want to run the tests

Reply:
The whole point of running it on desktop is to run in virtual display mode (that uses tk). I'll add some more clarification

Reply:
done